.github/elemental-iso-add-registration

```diff
@@ -25,7 +25,7 @@ abort() {
 }
 
 iso_repack() {
-  local TEMPDIR=$(mktemp -d)
+  local TEMPDIR=$(mktemp -d $HOME/elemental_iso.XXX)
```

Comment:
just wondering: why do we need this change? 🤔

Reply:
Mecause of MacOS, @juadk did some test and on mac it tries to create the temp directory in `/var/folders`... Don't know why... But this trick works both on Linux and Mac.
Maybe we should have add a comment here yes...